src/coreclr/jit/instrsxarch.h

```diff
@@ -1293,6 +1293,27 @@ INST1(setge,            "setge",            IUM_WR, 0x0F009D,
 INST1(setle,            "setle",            IUM_WR, 0x0F009E,                                                            ILLEGAL,           ILLEGAL,    INS_TT_NONE,    Reads_OF       | Reads_SF      | Reads_ZF | Encoding_REX2)
 INST1(setg,             "setg",             IUM_WR, 0x0F009F,                                                            ILLEGAL,           ILLEGAL,    INS_TT_NONE,    Reads_OF       | Reads_SF      | Reads_ZF | Encoding_REX2)
 
+#ifdef TARGET_AMD64
+// The following instructions shall always be next to SETcc instructions group, the offset between the original instruction and the ZU variant should be 16.
+// No new instruction should be inserted from INS_seto to setzug.
```

Comment:
I'm not sure this is particularly relevant. The instructions can be anywhere and the offset would be `INS_setzuo - INS_setz`. That is currently 16, but would remain valid regardless of where it was moved in the file.